docs/release-notes.7.0.md

```diff
@@ -1,5 +1,17 @@
 # Qooxdoo Release Notes
 
+## New features in v7.4
+- compile.json: localModules can now be used in the root of the file. So you can adress the modules for all apllications.
+                Application modules will be merged with global ones.
+- browserify of npm modules: Browserify of the used npm modules is a time cosuming process. No it's checked if it's really neccesary.
```

Comment:
s/Browserify of the/Broserifying the/
s/time cosuming/time-consuming/
s/No it's chechecked if/Now it's checked whether/